src/main/scala/com/fulcrumgenomics/alignment/Aligner.scala

```diff
@@ -512,7 +512,7 @@ class Aligner(val scorer: AlignmentScorer,
   private def findByScore(matrices: Array[AlignmentMatrix], minScore: Int): Seq[MatrixLocation] = {
     val qLen = matrices(0).queryLength
     val tLen = matrices(0).targetLength
-    val hits = new ArrayBuffer[MatrixLocation]()
```

Comment:
Can you explain this change for my own learning?  I would have expected `ArrayBuffer.newBuilder` (maybe)?  Why doesn't `new ArrayBuffer[]()` work anymore?  It looks like they're still here: https://docs.scala-lang.org/overviews/collections/concrete-mutable-collection-classes.html#array-buffers

Reply:
The problem stems from the fact that all the type aliases in `scala.` that are automatically available have been changed.  They be things like `type Seq[+A] = scala.collection.Seq[A]`, but they have been changed in 2.13, to point instead to the immutable version of the traits.  That leaves us with a choice between two routes:

1) Explicitly import `scala.collection.Seq` and shadow `scala.Seq` when we want to specify _either mutable or immutable_, mirroring the 2.12 behaviour. 
2) Change our code to produce immutable `Seq`s (and `IndexedSeq`s etc.) so that they are now of type `scala.Seq`.

Since we're generally treating `Seq` and `IndexedSeq` as immutable types, I thought it was better go with with option (2).  The problem this causes is that we used to get away with using things like `ArrayBuffer` and `ListBuffer` and rely on the fact that they implemented `Seq`, but they implement `mutable.Seq`.  So I _think_ that a call to `ArrayBuffer.toSeq` will cause a second builder to be made to make the immutable Seq.

I'm assuming that using the builders efficient, but haven't tested.  Note that this is what any `.map()`, `.filter()` etc uses to build the new collection, so it's not like we haven't been relying on them all along.